question-service/index.js

```diff
@@ -1,10 +1,34 @@
 import express from "express";
 import cors from "cors";
+import mongoose from "mongoose";
+import "dotenv/config";
+import questionRoutes from "./route/question-route.js";
+
+// const uri =
+//   process.env.ENV == "PROD"
+//     ? process.env.DB_CLOUD_URI
+//     : "mongodb://localhost:27017/";
+
+const uri = process.env.DB_CLOUD_URI;
```

Comment:
To discuss with the team the standardization of this statement. I would expect 2 possible URIs, as one would be for testing in CI.

Reply:
Understood, made the required changes.